core/relay/config/Corda_Relay2.toml

```diff
@@ -1,40 +1,52 @@
-name = "Corda_Relay"
-port="9080"
+name = "Corda_Relay2"
+port="9082"
 host="localhost"
 hostname="localhost"
 db_path="db/Corda_Relay2/requests"
 # This will be replaced by the task queue.
 remote_db_path="db/Corda_Relay2/remote_request"
 
+# FOR TLS
+cert_path="credentials/fabric_cert.pem"
+key_path="credentials/fabric_key"
+tls=false
+
 [networks]
-[networks.Corda_Network]
+[networks.Corda_Network2]
```

Comment:
As above.

Reply:
I don't get it, why does `Corda_Relay2` need to connect to `Corda_Network`'s driver instead of `Corda_Network2`'s driver?

Reply:
Right, this is OK. But the `relays` definitions above should list other relays and not the local, no?